crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -157,18 +163,34 @@ export function Errors({ errors }: ErrorsProps) {
     []
   );
 
+  const hasIssues = issues.length > 0;
+  const hasIssueWithError = issues.some((issue) =>
+    ["bug", "fatal", "error"].includes(issue.severity)
+  );
+
   const hasErrors = errors.length > 0;
   const hasServerError = readyErrors.some((err) =>
     ["server", "edge-server"].includes(getErrorSource(err.error) || "")
   );
-  const isClosable = !isLoading && !hasServerError;
+  const isClosable = !isLoading && !hasIssueWithError && !hasServerError;
+
+  const defaultTab =
+    hasIssueWithError || !hasErrors
+      ? TabId.TurbopackIssues
+      : TabId.RuntimeErrors;
 
-  const defaultTab = TabId.RuntimeErrors;
   const [selectedTab, setSelectedTab] = React.useState<string>(defaultTab);
 
+  React.useEffect(() => {
+    if (defaultTab === TabId.TurbopackIssues) {
+      setSelectedTab(TabId.TurbopackIssues);
+      setDisplayState("fullscreen");
+    }
+  }, [defaultTab]);
+
   // This component shouldn't be rendered with no errors, but if it is, let's
   // handle it gracefully by rendering nothing.
-  if (errors.length < 1) {
+  if (errors.length < 1 && issues.length < 1) {
```

Comment:
`=== 0` is clearer imo. `< 1` makes it look like the value can be negative, but we're really only checking for the empty case.